README.md

```diff
@@ -45,55 +48,69 @@ NOTE: If you would like to _improve_ the content of this repository, by fixing t
+3. *Classification challenge*: Classify a hex as formal, informal or rural based on the data derived from the service request data.
 
-In the `data` directory you will find a dataset file, `data/sr.csv`, which contains 36 months of service request data, where each row is a service request. A service request is a request from one of our residents to undertake significant work.
+Feel free to use any other data you can find in the public domain, except for task (3).
 
-We have made two resources available remotely:
-* A GeoJSON file that contains the level 8, 9 and 10 resolution hexagons for the City of Cape Town at [this location](insert readonly object url).
-* An AWS S3 writeonly bucket at [this location](Insert here).
+**The final output of the execution of your code should be a self-contained `html` file or executed `ipynb` file that is your report.** 
+ 
+A statistically minded layperson should be able to read this report and follow your analysis without guidance.https://h3geo.org/
 
-1. *Extraction and Load Challenge* - create a script which joins the SR data to the H3 hex data, meeting the following requirements:
-   1. Uses AWS S3 SELECT syntax to read in H3 resolution 8 data. 
-   2. Joins it to the service request dataset
-   3. Select a subset of columns (including the H3 index column), and write it to the writeonly bucket. Be sure to name your output file something that is recognizable as your work, and unlikely to collide with the names of others. 
-   * Include logging that lets the executor know how many of the records failed to join, and include a join error threshold above which the script will error out. 
+Please log the time taken to perform the operations described, and within reason, try to optimise latency and computation resources used.
 
-2. *Transformation challenge* - write a script which anonymises the SR file, but preserves the following resolutions (You may use H3 indexes or lat/lon coordinates for your spatial data):
+### 5. Further Data Transformations (if applying Data Engineering Positions)
+Write a script which anonymises the `sr.csv` file, but preserves the following resolutions (You may use H3 indexes or lat/lon coordinates for your spatial data):
    * location accuracy to within approximately 500m 
    * temporal accuracy to within 6 hours
    * scrubs any columns which may contain personally identifiable information.
- 
-Your code should be well formatted according to generally accepted style guides and include whatever is necessary for a team-mate unfamiliar with it to maintain it.
 
-## Contact
+We expect in the accompanying report that follows you will justify as to why this data is now anonymised.
```

Comment:
Please limit this commentary to less than 500 words. If your code is written in a code notebook such as Jupyter notebook or Rmarkdown, you can include the report in your notebook.